pom.xml

```diff
@@ -45,7 +45,9 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.13</slf4j.version>
```

Comment:
This isn't required.

Reply:
I know, I always update slf4j to the latest. But it can of course be skipped